### PR TITLE
Adding role readme for collection

### PR DESCRIPTION
**Pull request by hcherukuri**
Adding role readme for collection

### CODE REVIEW

File: roles/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Using the JCliff collection within your playbook
+
+Once the Collection has been installed and JCliff is available on the system, you can directly use the tool within your playbook to configure a EAP or Wildfly instance:
+
+    ---
+    - hosts: localhost
+      gather_facts: false
+      collections:
+        - wildfly.jcliff
```
Comment (sabre1041) [inline roles/README.md:55]: Please update this

File: roles/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      tasks:
+        - name: Include Jcliff role
+          include_role:
+            name: jcliff
+
+If you would like to use it on windows we have named the role "win-jcliff" that will install out jcliff collection on windows. We have created a separate role for windows environment.
+    - hosts: ...
+      collections:
+        - wildfly.jcliff
```
Comment (sabre1041) [inline roles/README.md:38]: Please update this

File: roles/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Ansible Collection - wildfly.jcliff
```
Comment (sabre1041) [inline roles/README.md:1]: Update this

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Comment by hcherukuri** [comment]
Changes done. Thank you for pointing them out.

**Comment by sabre1041** [comment]
@hcherukuri can you tackle the primary readme too?

**Comment by hcherukuri** [comment]
@sabre1041 I did it for both in the PR.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**